ui/src/App.js

```diff
@@ -84,12 +87,19 @@ export default () => (
             path={`${config.HOMEPAGE}/projects/:projectId/models/:modelId/versions/:versionId/*`}
             render={PrivateLayout(VersionDetails)}
           />
+
           <Redirect
             from={`${config.HOMEPAGE}/projects/:projectId/models/:modelId/versions/:versionId`}
-            to={`${config.HOMEPAGE}/projects/:projectId/models/:modelId/versions/:versionId/deploy`}
+            to={`${config.HOMEPAGE}/projects/:projectId/models/:modelId/versions/:versionId/endpoints/-/details`}
             noThrow
           />
 
+          {/* Version pages (and its sub-routers) */}
+          <PrivateRoute
+            path={`${config.HOMEPAGE}/projects/:projectId/models/:modelId/versions/:versionId/endpoints/:endpointId/*`}
```

Comment:
I think the path should be `/projects/:projectId/models/:modelId/versions/:versionId` since it's version details page and not endpoint